pikaur/build.py

```diff
@@ -389,12 +389,18 @@ def _set_built_package_path(self) -> None:
                 BuildError(_("{} does not exist on the filesystem.").format(pkg_path))
             if dest_dir == self.build_dir:
                 new_package_path = os.path.join(PACKAGE_CACHE_PATH, pkg_filename)
+                pkg_sig_path = pkg_path + ".sig"
+                new_package_sig_path = new_package_path + ".sig"
                 if os.path.exists(pkg_path):
                     if not os.path.exists(PACKAGE_CACHE_PATH):
                         os.makedirs(PACKAGE_CACHE_PATH)
                     if os.path.exists(new_package_path):
                         os.remove(new_package_path)
                     shutil.move(pkg_path, PACKAGE_CACHE_PATH)
+                if os.path.exists(pkg_sig_path):
+                    if os.path.exists(new_package_sig_path):
+                        os.remove(new_package_sig_path)
+                    shutil.move(pkg_sig_path, PACKAGE_CACHE_PATH)
```

Comment:
could you please factor out it as `replace_file()` in https://github.com/actionless/pikaur/blob/master/pikaur/core.py instead of copy-pasting code block?